awswrangler/_config.py

```diff
@@ -149,47 +157,82 @@ def to_pandas(self) -> pd.DataFrame:
         return pd.DataFrame(args)
 
     def _load_config(self, name: str) -> bool:
+        if _CONFIG_ARGS[name].is_parent:
+            if self._loaded_values.get(name) is None:
+                self._set_config_value(key=name, value={})
+                return True
+            return False
```

Comment:
I am not sure I understand this case. I currently read it as if the `parent` is not in `loaded_values`, then load it with an empty dict and return True. If it's already in loaded values then return False. Why False it's already loaded?

Reply:
I was under the impression that this function returns `True` if a value was set during the call, and `False` if there was no need for anything to be set. So I tried to make it consistent with the rest of the function. However, the return value isn't used anywhere so it doesn't seem to matter.

Is the return value meant to signify something different? Also, seeing as it's not used, should I simply change it to not return anything?